prog/analysis.go

```diff
@@ -21,6 +21,7 @@ type state struct {
 	strings   map[string]bool
 	ma        *memAlloc
 	va        *vmaAlloc
+	listFiles []string
```

Comment:
This also looks unnecessary since state already contains target, so the code can just use s.target.Files.